src/components/Settings/Settings.js

```diff
@@ -3,31 +3,77 @@ import FAB from "../FAB";
 import ButtonAppBar from "../AppBar";
 import Popup from "../Settings/Popup";
 import { useState } from "react";
-import { Button } from "@material-ui/core";
+import { Divider, List, ListItem, ListItemText, makeStyles, Typography } from "@material-ui/core";
 import ProfilePopup from "./ProfilePopup";
 import GroupLink from "./GroupLink";
+import Wrapper from "../base/Wrapper";
+import { navigate } from "@reach/router";
+const useStyles = makeStyles((theme) => ({
+  button: {
+    padding: theme.spacing(1, 0),
+  },
+  info: {
+    display: "flex",
+    alignItems: "center",
+    position: "fixed",
+    color: theme.palette.text.main,
+    bottom: 70,
+    right: theme.spacing(12),
+    zIndex: 2,
+  },
+}));
 
 function Settings() {
   const [openGroupPopup, setOpenGroupPopup] = useState(false);
   const [openEditProfile, setOpenEditProfile] = useState(false);
   const [tiggerUpdate, setTriggerUpdate] = useState(false);
   const [openGroupLink, setOpenGroupLink] = useState(false);
+  const classes = useStyles();
 
   return (
     <>
       <ButtonAppBar title="Einstellungen" />
       <Bubbles updateMe={tiggerUpdate} />
-      <FAB open={() => setOpenGroupPopup(true)} />
-      <h1>Settings</h1>
-      <Popup
-        open={openGroupPopup}
-        close={() => setOpenGroupPopup(false)}
-        updateBubbles={() => setTriggerUpdate(!tiggerUpdate)}
-      />
-      <Button onClick={() => setOpenEditProfile(true)}>Profil bearbeiten</Button>
-      <ProfilePopup open={openEditProfile} close={() => setOpenEditProfile(false)} />
-
-      <GroupLink open={openGroupLink} close={() => setOpenGroupLink(false)} />
+      <Wrapper>
+        <List component="nav" aria-label="settings options">
+          <Divider />
+          <ListItem button className={classes.button} onClick={() => setOpenEditProfile(true)}>
+            <ListItemText primary="Profil bearbeiten" />
+          </ListItem>
+          <Divider />
+          <ListItem
+            button
+            className={classes.button}
+            onClick={() => {
+              navigate("/impressum");
+            }}
+          >
+            <ListItemText primary="Impressum" />
+          </ListItem>
+          <Divider />
+          <ListItem
+            button
+            className={classes.button}
+            onClick={() => {
+              navigate("/datenschutz");
+            }}
+          >
+            <ListItemText primary="Datenschutz" />
+          </ListItem>
+          <Divider />
+        </List>
+        <Typography className={classes.info}>
```

Comment:
```suggestion
        <Typography className={classes.info} color="primary">
```
Finde ich in primary hübscher :)